packages/core/test/SchemaField_test.js

```diff
@@ -428,4 +428,128 @@ describe("SchemaField", () => {
       });
     });
   });
+  describe("hideError flag and errors", () => {
+    const schema = {
+      type: "object",
+      properties: {
+        foo: { type: "string" },
+      },
+    };
+
+    const uiSchema = {
+      "ui:hideError": true,
+      "ui:field": props => {
+        const { uiSchema, ...fieldProps } = props; //eslint-disable-line
+        return <SchemaField {...fieldProps} />;
+      },
+    };
```

Comment:
- Reuse the same `schema` variable from before
- `uiSchema = {...uiSchema, "ui:hideError": true};`

Reply:
done